src/raw_client.rs

```diff
@@ -1149,20 +1170,21 @@ mod test {
         assert_eq!(resp.hash_function, Some("sha256".into()));
         assert_eq!(resp.pruning, None);
     }
+
     #[test]
     fn test_relay_fee() {
         let client = RawClient::new(get_test_server(), None).unwrap();
 
-        let resp = client.relay_fee().unwrap();
-        assert_eq!(resp, 0.00001);
+        let resp = client.relay_fee().unwrap().to_sat_per_vb_ceil();
+        assert!(resp > 0);
```

Comment:
I probably need some time to take a better look at the tests to wrap my mind around it, I'm afraid we can be missing something due to the current state of testing at crate level 🤔